code/modules/mob/mob_emote.dm

```diff
@@ -28,12 +28,16 @@
 		return FALSE
 	var/silenced = FALSE
 	for(var/datum/emote/P in key_emotes)
-		if(!P.check_cooldown(src, intentional))
-			// if an emote's on cooldown, don't spam them with messages of not being able to use it
-			silenced = TRUE
+		// can this mob run the emote at all?
+		if(!P.can_run_emote(src, intentional))
 			continue
-		if(P.try_run_emote(src, param, type_override, intentional))
-			return TRUE
+		else
```

Comment:
That ```else``` is necessary?

Reply:
No, purely added for readability, can be removed if preferred.